.github/workflows/labeler.yml

```
@@ -1,7 +1,11 @@
 name: Process PRs
 
 on:
-  - pull_request_target
+  - pull_request:
```

actionlint (1.7.12)

[error] expected scalar node for string value but found mapping node with "!!map" tag

(syntax-check)